stdlib/3/time.pyi

```diff
@@ -5,10 +5,17 @@
 # see: http://nullege.com/codes/search?cq=time
 
 import sys
-from typing import Tuple, Union
+from typing import Any, NamedTuple, Tuple, Union
 from types import SimpleNamespace
 
-TimeTuple = Tuple[int, int, int, int, int, int, int, int, int]
+if sys.version_info >= (3, 3):
+    TimeTuple = Union[
```

Comment:
I see what you're saying. Two thoughts.

1. This looks like a possible future improvement to Mypy? I don't see the current behavior specified in PEP 484.
2. The main point of the TimeTuple type is checking arguments to functions (no such name in the actual `time` module). In this context, it works exactly as expected. See example below.

```
import time
print(time.struct_time((1,2)))
```
At runtime this fails with:
```
Traceback (most recent call last):
  File "/tmp/qwe.py", line 10, in <module>
    print(time.struct_time((1,2)))
TypeError: time.struct_time() takes an at least 9-sequence (2-sequence given)
```
With my change to time.pyi, Mypy correctly fails as well:
```
/tmp/qwe.py:10: error: Argument 1 to "struct_time" has incompatible type "Tuple[int, int]"; expected "Union[Tuple[int, int, int, int, int, int, int, int, int], Tuple[int, int, int, int, int, int, int, int, int, str], Tuple[int, int, int, int, int, int, int, int, int, str, int]]"
```

So, I'd file an issue against the current behavior of Mypy and accept the current union, esp. that time.pyi has no API that **returns** TimeTuples, it's just a function argument. It also works for type mismatches for tuple fields:
```
/tmp/qwe.py:2: error: Argument 1 to "struct_time" has incompatible type "Tuple[str, str, str, str, str, str, str, str, str]"; expected "Union[Tuple[int, int, int, int, int, int, int, int, int], Tuple[int, int, int, int, int, int, int, int, int, str], Tuple[int, int, int, int, int, int, int, int, int, str, int]]"
```